main/interdiff.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/golang/glog"
+	"github.com/google/go-patchutils"
+	"github.com/google/subcommands"
+)
+
+type interdiffCmd struct {
+	oldDiff string
+	newDiff string
+}
+
+func init() {
+	subcommands.Register(&interdiffCmd{}, "")
+}
+
+func (*interdiffCmd) Name() string { return "interdiff" }
+func (*interdiffCmd) Synopsis() string {
+	return "compute difference between " +
+		"source patched with oldDiff and same source patched with newDiff."
+}
+func (*interdiffCmd) Usage() string {
+	return "interdiff -olddiff=<oldDiff path> -newdiff=<newDiff path>: " +
+		"Compute difference between source patched with oldDiff and same source patched with newDiff."
+}
+
+func (i *interdiffCmd) SetFlags(f *flag.FlagSet) {
```

Comment:
the method receiver is usually called `c` as a shorthand for subCommand. This avoids naming conflicts e.g. if you'd want a loop with an index variable `i`.